app/views/pages/home.html.erb

```diff
@@ -59,3 +59,18 @@
         </div>
       </div>
     </div>
+
+
+<% content_for(:after_js) do %>
+ <script>
```

Comment:
Here you included the JS on the home page only. You have to choose between having it here OR in `app/assets/javascripts/doorbell.js`, but do not use both.